api/xml/xml_test.go

```diff
@@ -132,7 +132,10 @@ func TestPackage(t *testing.T) {
 	for _, test := range tests {
 		t.Run(test.Schema, func(t *testing.T) {
 
-			result, err := service.DefaultTemplate(test.Schema, test.Data)
+			app := api.BlankApplication(-1, "SF86", "2017-07")
```

Comment:
Can we add complete test scenario tests for the 85 and 85P? There are some already in the repo in subdirectories under `complete-scenarios`.

Reply:
(Not on this line of course – just used it as a spot to hang my comment on.)

Reply:
I tried this, there's enough places where XML is being inserted at the top level (see foreign.xml for instance) that trying to put the rules in the sub-sections ends up being redundant. I think I'm going to leave things as is rather than making it so sometimes you have to check the top level and sometimes you don't.